apps/voting-disputable/app/src/components/VoteDetail/DisputablePeriod.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
I see we are importing `prop-types` but it's not used, perhaps we should define them for this component? also worth checking whether the linting is configured correctly as I didn't see any errors for this and they should be being flagged.